swiggy-pdp-cart/src/content.js

```diff
@@ -0,0 +1,93 @@
+import React from "react";
+import Menu from "./menu";
+import Cart from "./cart";
+import Categories from "./categories";
+
+export default class Content extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleChange = this.handleChange.bind(this);
+    this.state = {
+      category: "recommended",
```

Comment:
Going forward, the initial state can grow into an object with many keys, possible nested.

Always keep initial state separately configured - this makes it more scalable and easier to maintain

```
const INITIAL_STATE = {
category: "recommended"
}
```